exchanges/bithumb/bithumb_test.go

```diff
@@ -510,170 +468,139 @@ func TestModifyOrder(t *testing.T) {
 		Pair:      curr,
 		AssetType: asset.Spot,
 	})
-	if err != nil {
-		t.Error(err)
-	}
+	require.NoError(t, err, "ModifyOrder must not error")
 }
 
 func TestWithdraw(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
-
-	withdrawCryptoRequest := withdraw.Request{
-		Exchange:    b.Name,
-		Amount:      -1,
-		Currency:    currency.BTC,
-		Description: "WITHDRAW IT ALL",
-		Crypto: withdraw.CryptoRequest{
-			Address: core.BitcoinDonationAddress,
-		},
-	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 
-	_, err := b.WithdrawCryptocurrencyFunds(context.Background(),
-		&withdrawCryptoRequest)
-	if !sharedtestvalues.AreAPICredentialsSet(b) && err == nil {
-		t.Error("Expecting an error when no keys are set")
-	}
-	if sharedtestvalues.AreAPICredentialsSet(b) && err != nil {
-		t.Errorf("Withdraw failed to be placed: %v", err)
-	}
+	t.Skip("TestWithdraw not allowed for live tests")
```

Comment:
What's happened here? This looks like it should be put back

Reply:
The test would fail if leaving the amount to -1. If we set it to a positive amount, then with the credentials being set and canManipulateRealOrders to true, the user can accidentally send the money to the donation address. This solution was suggested for the bitstamp_test, so I implemented it here too.